Bottom: 4bbcb14e0681aad0a350fedd37f6fa80dd36ffe4
Top:    93c07ff797e3d0c6e229fbb98fd9430e8a20b78c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 21:12:40 +0100

Add support for a two soldermask layers, defining regions of removed mask

File-format wise, these work similarly to how silk layers are supported now:

Layer (1 to n "[NAME]")     # "ordinary" design layers
Layer (n+1 "silk")          # top side
Layer (n+2 "silk")          # bottom side
Layer (n+3 "soldermask")    # top side
Layer (n+4 "soldermask")    # bottom side


---

diff --git a/src/action.c b/src/action.c
index 7a797f3..d91b9f5 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1386,7 +1386,9 @@ NotifyMode (void)
 		int flag = CLEARLINEFLAG;
 
 		if (GetLayerGroupNumberByNumber (INDEXOFCURRENT) ==
-		    GetLayerGroupNumberByNumber (bottom_silk_layer))
+		    GetLayerGroupNumberByNumber (bottom_silk_layer) ||
+		    GetLayerGroupNumberByNumber (INDEXOFCURRENT) ==
+		    GetLayerGroupNumberByNumber (bottom_soldermask_layer))
 		  flag |= ONSOLDERFLAG;
 		if ((text = CreateNewText (CURRENT, &PCB->Font, Note.X,
 					   Note.Y, 0, Settings.TextScale,
